### PR TITLE
Create a "Getting Started" Page in the Docs Vault and the Backend Solution Readme

### LINKED ISSUE #1: Create a "Getting Started" Page in the Docs Vault and the Backend Solution Readme
## Description
Create a “Getting Started” page/section which describes a way in which anyone can get the backend solution running on a local dev setup. 

Note that this does not involve firing up an IDE, opening the project and running from therein – instead, one should clone the Timelines repository, and then spin up the Docker assembly.

Take available aliases into account, but do not insist on them as those are optional.

Pay attention to proper English – use ChatGPT for corrections if needed, but proof-read the result.

## Acceptance criteria
- A “Getting Started” page is available in the Docs Obsidian vault of the app repo.
- A “Getting Started” section is added to the README.md of the backend solution.
- Test the tutorial on Windows and WSL. Successful test should include requests to `GET /health` and `GET /Nodes` and the subsequent “healthy” responses. NB: Use Docker ports when testing.

## Assessment
Story points: 3

### PR DESCRIPTION
**Pull request by HunorTotBagi**
### The following scenarios were tested:
1. Windows with aliases - Working
2. Windows without aliases - `source` command is not recognized since it’s Linux thingy
3. WSL with aliases - `source` command feeds the aliases into the terminal but for example `tlupb` returns error message `/Timelines/Backend/src/various/DockerCompose/docker-compose.yml: no such file or directory`. Clearly the `DockerCompose` folder is not in `Various` folder
4. WSL without aliases - Working

### In working scenarios the following results were obtained:

`http://localhost:26001` produced `Hello World! This is Timelines.` result

`http://localhost:26001/health` returned status `Healthy` in a json format

`http://localhost:26001/Nodes` gave message `Hello World! This is Timelines Nodes module.`

Closes #1

### CODE REVIEW

File: Backend/README.md [hunk 1]
```diff
@@ -30,3 +30,47 @@ From the solution root run `source ./Various/aliases` to source the following al
...
+```
+docker compose -f ./DockerCompose/docker-compose.yml -f ./DockerCompose/docker-compose.override.yml up --build -d
+```
+
+### Step 4: Verify the Setup
+
+To confirm the containers are running correctly, open your browser and visit the following URLs:
+
+- [http://localhost:26001](http://localhost:26001)
+- [http://localhost:26001/health](http://localhost:26001/health)
```
Comment (NikolaVetnic) [inline Backend/README.md:74]: This one should be `GET /Nodes` to test the other one.

**Review by NikolaVetnic — CHANGES_REQUESTED** [review]

File: Backend/README.md [hunk 2]
```diff
@@ -30,3 +30,47 @@ From the solution root run `source ./Various/aliases` to source the following al
...
+
+## 4. Getting Started
+
+Follow the steps below to set up and run the project:
+### Step 1: Clone the Repository
+
+Open your terminal and execute the following command to clone the repository:
+
+```
+git clone https://github.com/NikolaVetnic/Timelines.git
```
[the remark this answers is not in the document]
Comment (HunorTotBagi) [reply Backend/README.md:42]: Done

File: Backend/README.md [hunk 3]
```diff
@@ -30,3 +30,47 @@ From the solution root run `source ./Various/aliases` to source the following al
...
+
+### Step 2: Load Predefined Aliases (Optional)
+
+Navigate to the `./Various` directory and load the predefined aliases into your terminal by running:
+
+```
+source aliases
+```
+
+This step configures your terminal with convenient shortcuts. For instance, the alias `tlupb` (short for _Timelines Up Build_) automates pulling the necessary Docker images and setting up the containers.
```
[the remark this answers is not in the document]
Comment (HunorTotBagi) [reply Backend/README.md:53]: Done

File: Backend/README.md [hunk 4]
```diff
@@ -30,3 +30,47 @@ From the solution root run `source ./Various/aliases` to source the following al
...
+
+### Step 3: Start the Containers
+
+To start the containers, use the alias:
+
+```
+tlupb
```
[the remark this answers is not in the document]
Comment (HunorTotBagi) [reply Backend/README.md:60]: Done

File: Backend/README.md [hunk 5]
```diff
@@ -30,3 +30,47 @@ From the solution root run `source ./Various/aliases` to source the following al
...
+
+Alternatively, if you prefer not to use aliases, navigate to the `/Backend/src` directory and run the equivalent command manually:
+
+```
+docker compose -f ./DockerCompose/docker-compose.yml -f ./DockerCompose/docker-compose.override.yml up --build -d
+```
+
+### Step 4: Verify the Setup
+
+To confirm the containers are running correctly, open your browser and visit the following URLs:
```
[the remark this answers is not in the document]
Comment (HunorTotBagi) [reply Backend/README.md:71]: Done

File: Backend/README.md [hunk 6]
```diff
@@ -30,3 +30,47 @@ From the solution root run `source ./Various/aliases` to source the following al
...
+
+Open your terminal and execute the following command to clone the repository:
+
+```
+git clone https://github.com/NikolaVetnic/Timelines.git
+```
+
+### Step 2: Load Predefined Aliases (Optional)
+
+Navigate to the `./Various` directory and load the predefined aliases into your terminal by running:
```
[the remark this answers is not in the document]
Comment (HunorTotBagi) [reply Backend/README.md]: Done

File: Backend/README.md (hunk 1 above)
Comment (HunorTotBagi) [reply Backend/README.md]: Done

File: Backend/README.md [hunk 7]
```diff
@@ -30,3 +30,47 @@ From the solution root run `source ./Various/aliases` to source the following al
...
+
+```
+docker compose -f ./DockerCompose/docker-compose.yml -f ./DockerCompose/docker-compose.override.yml up --build -d
+```
+
+### Step 4: Verify the Setup
+
+To confirm the containers are running correctly, open your browser and visit the following URLs:
+
+- [http://localhost:26001](http://localhost:26001)
```
[the remark this answers is not in the document]
Comment (HunorTotBagi) [reply Backend/README.md]: Done

File: Backend/README.md [hunk 8]
```diff
@@ -1,8 +1,43 @@
...
+```
+docker compose -f ./DockerCompose/docker-compose.yml -f ./DockerCompose/docker-compose.override.yml up --build -d
+```
+
+### 1.4 Verify the Setup
+
+Once the containers are running the following `Core.Api` endpoints will be available on `localhost:26001`
+
+- `GET /health`
+- `GET /Nodes`
```
Comment (NikolaVetnic) [inline Backend/README.md:39]: Empty line here pls.

File: Backend/README.md [hunk 9]
```diff
@@ -1,8 +1,43 @@
...
+
+After navigating to the `/Backend/src` directory in the previous step, you can optionally load the predefined aliases by executing the following command:
+
+```
+source ./Various/aliases
+```
+
+Consult the [table given below](#4-Various) for the list of available aliases.
+
+### 1.3 Start the Containers
```
Comment (NikolaVetnic) [inline Backend/README.md:26]: I would merge 1.2 and 1.3 into a single section called "Start the Containers" containing the "Navigate to the `/Backend/src` directory and run the following" sentence.

In the next paragraph I would say that the aliases are available for Linux and MacOS users, and then I would give the command and path from which to source them.

**Review by NikolaVetnic — CHANGES_REQUESTED** [review]

File: Backend/README.md (hunk 8 above)
Comment (HunorTotBagi) [reply Backend/README.md:39]: Done

File: Backend/README.md (hunk 9 above)
Comment (HunorTotBagi) [reply Backend/README.md]: Done

**Review by NikolaVetnic — APPROVED** [review]

**Pull request merged**